### PR TITLE
Docs: remove/warn about references to FOSUSerBundle.

### PR DESCRIPTION
**Pull request by m-ar-c**
## Subject

Update documentation because `FOSUSerBundle` is not used by `SonataUserBundle` anymore.

I am targeting this branch, because the change is BC.

### CODE REVIEW

**Comment by VincentLanglet** [comment]
HI, I removed the changelog, there is no need for doc update

File: docs/reference/security.rst [hunk 1]
```diff
@@ -6,14 +6,13 @@ User management
 
 By default, the SonataAdminBundle does not come with any user management,
 however it is most likely the application requires such a feature. The Sonata
-Project includes a ``SonataUserBundle`` which integrates the ``FOSUserBundle``.
+Project includes a ``SonataUserBundle``.
```
Comment (VincentLanglet) [inline docs/reference/security.rst]: Maybe we can do better like
```
By default, the SonataAdminBundle does not come with any user management,
however it is most likely the application requires such a feature. For this, you can try
the ``SonataUserBundle``.
```

File: docs/reference/security.rst [hunk 2]
```diff
@@ -252,6 +251,11 @@ And specify it as Sonata security handler on your configuration:
 ACL and FriendsOfSymfony/UserBundle
 -----------------------------------
 
+.. note::
```
Comment (jordisala1991) [inline docs/reference/security.rst]: To me we should remove all references to FOSUserBundle, and replace with references to SonataUserBundle when needed.

References to the install of SonataUserBundle should be pointed to the SonataUserBundle docs.

File: docs/reference/security.rst [hunk 3]
```diff
@@ -252,6 +251,11 @@ And specify it as Sonata security handler on your configuration:
...
+
+   Some parts of this section are obsolete : ``SonataUserBundle`` does not use
+   ``FOSUserBundle`` anymore.
+
```
Comment (VincentLanglet) [inline docs/reference/security.rst]: Let's update the doc instead of adding a note :)

**Review by VincentLanglet — CHANGES_REQUESTED** [review]

**Comment by m-ar-c** [comment]
I think I did all that was suggested here. I have almost zero knowledge and experience with the things I edited, so please review my contribution carefully :)

File: docs/reference/security.rst [hunk 4]
```diff
@@ -339,22 +311,22 @@ and then configure SonataAdminBundle:
                 acl_user_manager: App\Manager\AclUserManager
                 # ...
 
-In your ``config/packages/fos_user.yaml`` you then need to put the following:
+In your ``config/packages/sonata_user.yaml`` you then need to put the following:
```
Comment (jordisala1991) [inline docs/reference/security.rst]: I think this whole block can be removed

Comment (m-ar-c) [reply docs/reference/security.rst]: Sorry I'm not sure what block you're talking about. I should remove line 314 to 325 (the `config/packages/sonata_user.yaml` code block) ?

Comment (jordisala1991) [reply docs/reference/security.rst]: Yes, exactly! Can you do it?

Comment (m-ar-c) [reply docs/reference/security.rst]: Np, I'll do it tomorrow or this week-end.

**Review by VincentLanglet — DISMISSED** [review]

File: docs/reference/security.rst [hunk 5]
```diff
@@ -738,15 +697,15 @@ User list customization
...
+``SonataUserBundle``.
 
 To customize displayed user override
 ``Sonata\AdminBundle\Controller\CRUDController::getAclUsers()``. This method must
 return an iterable collection of users::
 
     protected function getAclUsers(): \Traversable
     {
-        $userManager = $container->get('fos_user.user_manager');
+        $userManager = $container->get('sonata_user.user_manager');
```
Comment (jordisala1991) [inline docs/reference/security.rst]:
```suggestion
        $userManager = $container->get('sonata.user.manager.user');
```

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thank you @m-ar-c

**Pull request merged**